lib/Analysis/Utility.cpp

```diff
@@ -149,13 +149,34 @@ bool supportMMA(triton::DotOp op, int version) {
 }
 
 #ifdef USE_ROCM
+static bool supportMFMAGranularity(int m, int n, int k) {
+  // these limitations are dtype dependent, in future we may relax them
+  int granularityMN = 32;
```

Comment:
Should we define these as constants somewhere so we can change easily if needed?

Reply:
Right, this should be a constant.

About place: I actually can not think of better place for this constants.

P.s. If you are worried about duplication in C++ and python code: I want to refactor this and remove python part eventually, so there will be only one place with these constants.